hr_citizenship/models/hr_employee.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import fields, models
+
+
+class HrEmployee(models.Model):
+    _inherit = "hr.employee"
+
+    citizenship_ids = fields.One2many(
+        comodel_name="res.citizenship",
+        inverse_name="employee_id",
+        string="Multi citizenship",
+    )
+    identification_id = fields.Char(
```

Comment:
I would mention that the following fields are overloaded, it helps to understand that the standard behavior has changed.